components/kyma-environment-broker/internal/config/validator.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+const requiredFields = "additional-components"
```

Comment:
add a comment that it is a comma separated list of required fields

Reply:
good idea, I will add appropriate comment